fvtest/compilertriltest/CompareTest.cpp

```diff
@@ -379,3 +379,92 @@ INSTANTIATE_TEST_CASE_P(CompareTest, UInt64Compare, ::testing::Combine(
         std::make_tuple("lucmpge", lucmpge),
         std::make_tuple("lucmplt", lucmplt),
         std::make_tuple("lucmple", lucmple) )));
+
+int32_t ificmpeq(int32_t l, int32_t r) {
+    return (l == r) ? 123 : -456;
```

Comment:
I like this approach!

Instead of hardcoding the number everywhere, can we use some defined constants:

```c++
static const int32_t TRUE_NUM = 123;
static const int32_t FALSE_NUM = -456;

/* ... */

int32_t ificmpeq(int32_t l, int32_t r) {
    return (l == r) ? TRUE_NUM : FALSE_NUM;
}
```

:question: